pkg/component/input_spec.go

```diff
@@ -21,8 +21,9 @@ type InputSpec struct {
 	Shippers       []string    `config:"shippers,omitempty" yaml:"shippers,omitempty"`
 	Runtime        RuntimeSpec `config:"runtime,omitempty" yaml:"runtime,omitempty"`
 
-	Command *CommandSpec `config:"command,omitempty" yaml:"command,omitempty"`
-	Service *ServiceSpec `config:"service,omitempty" yaml:"service,omitempty"`
+	Command   *CommandSpec `config:"command,omitempty" yaml:"command,omitempty"`
+	Service   *ServiceSpec `config:"service,omitempty" yaml:"service,omitempty"`
+	SkipGroup bool         `config:"skip_group,omitempty" yaml:"skip_group,omitempty"`
```

Comment:
Not a fan of the naming of `skip_group`. How about `isolate_units: true`?

Reply:
Renamed and changed the terminology across the code.